unreleased/TEMPLATE.yaml

```diff
@@ -0,0 +1,11 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type:
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component:
+
+# A single sentence describing the change
```

Comment:
```suggestion
# A brief description of the change
```
Sometimes a few sentences are necessary for the note (deprecations especially come to mind; they are normally formatted as "X is dprecated.  Use Y instead).

Reply:
Makes sense.